app/ts/components/Submit.tsx

```diff
@@ -275,7 +275,7 @@ export const Submit = ({
 					<div>
 						{ bouquetNetwork.value.relayMode === 'mempool' ? <>
 								<div style = 'padding-bottom: 10px;'>
-									<SingleNotice variant = 'warn' title = 'Mempool mode is dangerous' description = { `You are currently using Mempool mode. When mempool mode is enabled. The transactions are sent as individual transactions. This means it's possible that only one of the transactions might end up on the chain. Use this mode only if a relay is not available for the network.`} />
+									<SingleNotice variant = 'warn' title = 'Mempool mode is dangerous' description = { `You are currently using Mempool mode. Transactions are sent individually, as a result, some transactions may not make it onto the blockchain. This mode should only be used if a relay is unavailable for the network. Additionally, if a sweeper is active on your account, there is a high risk that rescue attempts may fail, allowing the sweeper to steal your gas funds and other assets. Use this mode only as a last resort when no other options are available.`} />
```

Comment:
```suggestion
									<SingleNotice variant = 'warn' title = 'Mempool mode is dangerous' description = { `You are currently using Mempool mode. Transactions are sent individually so some transactions may not make it onto the blockchain. This mode should only be used if a relay is unavailable for the network. Additionally, if a sweeper is active on your account there is a high risk that rescue attempts may fail, allowing the sweeper to steal your gas funds and other assets. Use this mode only as a last resort when no other options are available.`} />
```

Reply:
Why isn't this text shared with the above text so they match (e.g., constant)?

Reply:
its almost the same, the start is bit different